src/Sylius/Behat/Context/Setup/UserContext.php

```diff
@@ -80,4 +80,24 @@ public function thereIsUserIdentifiedBy($email, $password)
         $this->sharedStorage->setCurrentResource('user', $user);
         $this->userRepository->add($user);
     }
+
+    /**
+     * @When I delete the account of :email
```

Comment:
We may use transformers to transform `:userEmail` to User entity and assert that given user exists at that moment.